src/main/java/org/medicmobile/webapp/mobile/SimpleJsonClient2.java

```diff
@@ -37,6 +39,7 @@ public JSONObject get(String url) throws JSONException, IOException {
 		return get(new URL(url));
 	}
 
+	@SuppressWarnings("PMD.CloseResource")
```

Comment:
I think the only thing wrong here is that PMD is not smart enough to realize that the `closeSafely` method is actually closing these resources.  The [`CloseResource`](https://pmd.github.io/latest/pmd_rules_java_errorprone.html#closeresource) PMD rule actually allows for configuring this to let it know that custom methods handle the closing.  I think we just need to add this to our `pmd.xml`: 

```.xml
  <rule ref="category/java/errorprone.xml/CloseResource">
    <properties>
      <property name="closeTargets" value="closeSafely" />
    </properties>
  </rule>
```

Reply:
Thanks! done